lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/nested-journey-definitions.yaml

```diff
@@ -458,3 +451,31 @@ KBVS:
           exitEventToEmit: next
         enhanced-verification:
           exitEventToEmit: enhanced-verification
+    PRE_EXPERIAN_PAGE:
+      response:
+        type: page
+        pageId: page-pre-experian-kbv-transition
+      events:
+        next:
+          targetState: CRI_EXPERIAN_KBV
+    PRE_DWP_KBV_PIP_PAGE:
+      response:
+        type: page
+        pageId: personal-independence-payment
+      events:
+        next:
+          targetState: PRE_DWP_KBV_TRANSITION_PAGE
+        end:
+          targetState: CRI_NINO
+          checkIfDisabled:
+            hmrcKbv:
+              targetState: PRE_EXPERIAN_PAGE
+    PRE_DWP_KBV_TRANSITION_PAGE:
+      response:
+        type: page
+        pageId: page-pre-dwp-kbv-transition
+      events:
+        next:
+          targetState: CRI_DWP_KBV
+        end:
+          exitEventToEmit: end
```

Comment:
Could we make this event name a bit more descriptive? e.g. `prove-another-way`

Reply:
I used `end` to keep consistency with how its done elsewhere and thats the event which is based from the frontend